parts/kubernetesmastercustomdata.yml

```diff
@@ -317,11 +185,19 @@ write_files:
     [Service]
     Restart=always
     ExecStartPre=/bin/mkdir -p /var/lib/kubelet
+    ExecStartPre=/bin/mount --bind /var/lib/kubelet /var/lib/kubelet
```

Comment:
Sure, I'm fine with that. I was more specifically asking about... why do you bind mount `/var/lib/kubelet`? Is that needed? Do other setups do this?